Source/WebCore/rendering/svg/RenderSVGResourceMarker.cpp

```diff
@@ -94,21 +96,16 @@ FloatPoint RenderSVGResourceMarker::referencePoint() const
 
 float RenderSVGResourceMarker::angle() const
 {
-    float angle = -1;
-    if (markerElement().orientType() == SVGMarkerOrientAngle)
-        angle = markerElement().orientAngle().value();
-
-    return angle;
+    return markerElement().orientAngle().value();
 }
 
 AffineTransform RenderSVGResourceMarker::markerTransformation(const FloatPoint& origin, float autoAngle, float strokeWidth) const
 {
-    float markerAngle = angle();
     bool useStrokeWidth = markerElement().markerUnits() == SVGMarkerUnitsStrokeWidth;
 
     AffineTransform transform;
     transform.translate(origin);
-    transform.rotate(markerAngle == -1 ? autoAngle : markerAngle);
+    transform.rotate(markerElement().orientType() == SVGMarkerOrientAngle ? angle() : autoAngle);
```

Comment:
If angle returns `std::optional<float>` this statement can be simplified like this:
```
    transform.rotate(angle().value_or(autoAngle));
```

Reply:
Done.